src/utils/formatTickPrice.ts

```diff
@@ -20,5 +20,5 @@ export function formatTickPrice({ price, atLimit, direction, placeholder, number
     return placeholder
   }
 
-  return formatPrice(price, numberType ?? NumberType.TokenNonTx)
+  return formatPrice({ price, type: numberType ?? NumberType.TokenNonTx })
```

Comment:
this doesn't take in locale which i believe is why on the TDP the price shown in the chart isn't converted
![Screenshot 2023-09-14 at 10 54 55 AM](https://github.com/Uniswap/interface/assets/11512321/f8c5d7dd-4e67-4f4e-93e3-8d4fac4163f5)